google-http-client/src/main/java/com/google/api/client/http/HttpResponseException.java

```diff
@@ -172,9 +199,9 @@ public Builder(HttpResponse response) {
       this(response.getStatusCode(), response.getStatusMessage(), response.getHeaders());
       // content
       try {
-        content = response.parseAsString();
-        if (content.length() == 0) {
-          content = null;
+        InputStream inputStream = response.getContent();
+        if (inputStream != null) {
```

Comment:
Please add a comment about the fact that we are reading the input stream so that the connection closes.

Reply:
Added comment.